### ISSUE TITLE
What did you change for 'xlm-r-distilroberta-base-paraphrase-v1' training?

### ISSUE THREAD

**Issue by PhilipMay** [UKPLab/sentence-transformers#509]
Hi,
what did you change for `xlm-r-distilroberta-base-paraphrase-v1` training compared to `xlm-r-bert-base-nli-stsb-mean-tokens`?
My tests show that it performs better on the german translated stsb:

``` python
model = SentenceTransformer('xlm-r-bert-base-nli-stsb-mean-tokens')  #Spearman: 0.8181
model = SentenceTransformer('xlm-r-distilroberta-base-paraphrase-v1')  # Spearman: 0.8201
```

**Comment by nreimers** [comment]
Hi @PhilipMay 
A paper is upcoming for the paraphrase models.

These models were trained on various datasets with Millions of examples for paraphrases, mainly derived from Wikipedia edit logs, paraphrases mined from Wikipedia and SimpleWiki, paraphrases from news reports, AllNLI-entailment pairs with in-batch-negative loss etc.

In internal tests, they perform much better than the NLI+STSb models as they have see more and broader type of training data. NLI+STSb has the issue that they are rather narrow in their domain and do not contain any domain specific words / sentences (like from chemistry, computer science, math etc.). The paraphrase models has seen plenty of sentences from various domains. 

More details with the setup, all the datasets, and a wider evaluation will follow soon.

**Comment by PhilipMay** [comment]
Great thanks. I am looking forward to read this paper.
So the `xlm-r-distilroberta-base-paraphrase-v1` did never see any stsb data?

**Comment by nreimers** [comment]
Correct, it never saw any STSb data. Just positive pairs (sent1, sent2) with sent1 and sent2 paraphrases of each other.

**Comment by PhilipMay** [comment]
Very nice. And that made it perform better on sentiment similarity for german sentences...
Closing it now.

**Comment by PhilipMay** [comment]
Can you please tell me which model did you use as the base for `xlm-r-distilroberta-base-paraphrase-v1`?
This one?
https://huggingface.co/xlm-roberta-base

**Comment by nreimers** [comment]
Yes

**Comment by PhilipMay** [comment]
Ok thanks. Closing again.

**Comment by PhilipMay** [comment]
> Hi @PhilipMay
> A paper is upcoming for the paraphrase models.
> 
> These models were trained on various datasets with Millions of examples for paraphrases, mainly derived from Wikipedia edit logs, paraphrases mined from Wikipedia and SimpleWiki, paraphrases from news reports, AllNLI-entailment pairs with in-batch-negative loss etc.
> 
> In internal tests, they perform much better than the NLI+STSb models as they have see more and broader type of training data. NLI+STSb has the issue that they are rather narrow in their domain and do not contain any domain specific words / sentences (like from chemistry, computer science, math etc.). The paraphrase models has seen plenty of sentences from various domains.
> 
> More details with the setup, all the datasets, and a wider evaluation will follow soon.

Hey @nreimers - what about the paper? Is it released?
Thanks
Philip

**Comment by nreimers** [comment]
@PhilipMay  Sadly not yet, it is rather far down in the paper pipeline :/

**Comment by PhilipMay** [comment]
Hey @nreimers 
could you please tell us some more about the datasets you used for xlm-r-distilroberta-base-paraphrase-v1?
If they are open maybe with a link? I am especialy interested in english and german datasets.
Thanks
Philip

**Comment by nreimers** [comment]
Hi @PhilipMay 
You can find the list of datasets here:
https://github.com/UKPLab/sentence-transformers/issues/747#issuecomment-776993279
